inc/extra_unity.h

```diff
@@ -0,0 +1,5 @@
+#pragma once
+#include "status.h"
+#include "unity.h"
+
+#define TEST_ASSERT_OK(code) TEST_ASSERT_EQUAL(STATUS_CODE_OK, (code))
```

Comment:
I didn't want to add this directly into unity and it will likely be used often so I wasn't sure where to put it. Do you have a suggestion as to a better location?